Documentation/devel/stage1-implementors-guide.md

```diff
@@ -50,10 +50,10 @@ An alternative stage 1 could forego systemd-nspawn and systemd altogether, or re
 All that is required is an executable at the place indicated by the `coreos.com/rkt/stage1/run` entrypoint that knows how to apply the pod manifest and prepared ACI file-systems to good effect.
 
 The resolved entrypoint must inform rkt of its PID for the benefit of `rkt enter`.
-Stage 1 must write the host PIDs of the pod's process #1 and that process's parent to these two files, respectively:
+Stage 1 implementors have two options for doing so; only one must be implemented:
```

Comment:
It was waiting for a child process of lkvm. But that child process does not exist.

I think that "only" is better. If both "pid" and "ppid" exist, which one is stage0 supposed to use?